nav2_behavior_tree/plugins/action/compute_path_to_pose_action.cpp

```diff
@@ -40,20 +40,23 @@ void ComputePathToPoseAction::on_tick()
 BT::NodeStatus ComputePathToPoseAction::on_success()
 {
   setOutput("path", result_.result->path);
+  setOutput("global_planner_result_code", result_.result->error_code);
   return BT::NodeStatus::SUCCESS;
 }
 
 BT::NodeStatus ComputePathToPoseAction::on_aborted()
 {
   nav_msgs::msg::Path empty_path;
   setOutput("path", empty_path);
+  setOutput("global_planner_error_code", result_.result->error_code);
```

Comment:
Aborted I think is the only error state here

Reply:
No, I think `aborted` is the only state whre the error code is meaningfully populated. Cancel means we called it so there's not really an error. Success means it came back with a result completed. 

The current status of this PR actually sets aborted to 0 and cancel to value, when it should be the opposite.

Also, please add inline comments when you set to zero why we do that for each of the cases (success is.. successful; cancel is user cancelled so algorithm didn't fail)

Reply:
Ah, 

I misunderstood you the first time :).